src/app/src/util/util.js

```diff
@@ -523,3 +524,5 @@ export const addProtocolToWebsiteURLIfMissing = (url) => {
 
 export const convertFeatureFlagsObjectToListOfActiveFlags = featureFlags =>
     keys(pickBy(featureFlags, identity));
+
+export const checkWhetherUserHasDashboardAccess = partialRight(get, 'is_superuser', false);
```

Comment:
This is clever, but I'm not convinced that it is an improvement over an arrow function.

```js
export const checkWhetherUserHasDashboardAccess = partialRight(get, 'is_superuser', false);
```

```js
export const checkWhetherUserHasDashboardAccess = user => get(user, 'is_superuser', false);
```

Reply:
I'll switch this to use the arrow function before merging.